src/SubmitForm/index.js

```diff
@@ -4,6 +4,8 @@ function run(options) {
 	$.nette.ext("submitForm", {
 		before: function (xhr, settings) {
 			if (settings.nette) {
+				let toEnabled = settings.nette.form.find('button:enabled');
+				settings.nette.form.data('toEnabled', toEnabled);
```

Comment:
tak tady bych to asi udelal orovnou settings.nette.form.data('enabledButtons', settings.nette.form.find('button:enabled'));